src/main/java/com/example/Main.java

```diff
@@ -1,11 +1,14 @@
 package com.example;
 
 public class Main {
+
+    static String greetingMessage = "Hello world!! How are you?";
```

Comment:
Wouldn't it be better to have this as a constant?

Reply:
You're right. I will change it.

Reply:
@Argon28 Done.